package.json

```diff
@@ -65,8 +65,8 @@
     "htmlWhitespaceSensitivity": "strict"
   },
   "engines": {
+    "node": "^20.0.0",
```

Comment:
I believe we should maintain the Node engine at version `18.x.x`. I noticed that you have already defined `pkgs.nodejs_18` in `flake.nix`.

Reply:
Correct, I didn't mean to push that file.